### ISSUE TITLE
[question] How do I ask a question?

### ISSUE THREAD

**Issue by mrolle45** [python/mypy#12426]
I see that an Issue can have various labels, including (question).
I don't see any way for me to create an issue with the (question) label.
I gather that GitHub won't allow non-contributors to assign labels because an inexperienced user could mess things up with an inappropriate label.
Could you add an item to your https://github.com/python/mypy/issues/new/choose page for this purpose?
I don't find the mypy Gitter useful because (1) it is broader in scope than just mypy, and (2) I want my question to show up on the python/mypy/issues page, just like the issue I am writing now.

**Comment by erictraut** [comment]
Maybe it would make sense to enable the "Discussions" tab? This feature was added to github about a year ago, and it works well for general questions.

**Comment by JukkaL** [comment]
I don't have much experience with the Discussions tab, but it could be a good option. On the other hand, discussions might get a bit fragmented, if there's Gitter, https://github.com/python/typing/discussions and mypy discussions -- but maybe that's okay.

**Comment by 97littleleaf11** [comment]
@JukkaL Maybe we can have a try on mypy discussion, which serves as a platform for mypy-related questions, such as configuration, plugins, etc. Gitter is hard to search and archive general questions.

**Comment by mrolle45** [comment]
> @JukkaL Gitter is hard to search and archive general questions.

I agree.  I asked a question on Gitter, but it was hard to find replies to it.
